layouts/shortcodes/dbm-postgres-agent-config-examples.md

```diff
@@ -0,0 +1,87 @@
+### One agent connecting to multiple hosts
+To connect to multiple hosts, create an entry for each host in the Postgres integration config.
+```yaml
+init_config:
+instances:
+  - dbm: true
+    host: localhost_A
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+  - dbm: true
+    host: localhost_B
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+  - dbm: true
+    host: localhost_C
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    [...]
+```
+
+### Running custom queries
+To collect custom metrics, use the `custom_queries` option. See the sample [postgres.d/conf.yaml](https://github.com/DataDog/integrations-core/blob/master/postgres/datadog_checks/postgres/data/conf.yaml.example) for more details.
+```yaml
+init_config:
+instances:
+  - dbm: true
+    host: localhost
+    port: 5432
+    username: datadog
+    password: '<PASSWORD>'
+    custom_queries:
+    - metric_prefix: ddprefix
+      query: SELECT age, salary, name FROM foo;
+      columns:
+        - name: foo_age
+          type: gauge
+        - name: foo_salary
+          type: count
+        - name: foo_name
+          type: tag
+      tags:
+        - query:custom
+```
+
+### Monitoring multiple databases 
+To specify the name of the database to connect to, use `dbname`. If omitted, the default system `postgres` database is queried. Additionally, configure `dbstrict` to `true` if you only want to gather metrics from the database provided in the `dbname` option.
```

Comment:
This section needs to be more opinionated. This is the most confusing aspect of our configuration. It should clearly explain to the customer how they should configure it, what they get by doing so, and why. 

```Additionally, configure `dbstrict` to `true` if you only want to gather metrics``` puts too much burden on the customer to understand the highly confusing options. These examples should be copy-paste-replace and work without any thought on the customer's part.